### PR TITLE
Personal Portfolio

### PR DESCRIPTION
**Pull request by ZohalMirkhel**
This pull request includes several updates and enhancements to the portfolio project to improve the overall user experience and design. The following changes have been implemented:

**CSS Enhancements:**

- Improved styling for the navigation bar, hero section, about me section, education section, and projects section.

- Added a smooth scroll behavior for a better navigation experience.

- Applied responsive design principles to mobile and PC.

- Enhanced the visual appeal with better color schemes, fonts, and layout adjustments.

### CODE REVIEW

**Comment by ZohalMirkhel** [comment]
@walwalasafi, please take a look at my project, thank you! 
It does have linters erorr, but I will work on that later.

**Comment by WALWALASAFI** [comment]
Dear @ZohalMirkhel, 
 i have checked your projects, 
1: it has linter errors, that u will fix it as soon as possible.
2: it doesnot have read me file.
3: it doesnot have demo link to have look to ur project.

**Comment by ZohalMirkhel** [comment]
Dear @WALWALASAFI, please take a look at my project, I believe I am done with the desktop version of the website, except for the Show-Less and More button which still has some problems. Do let me know if there is anything else that I have to add or bring any changes.

**Comment by WALWALASAFI** [comment]
@ZohalMirkhel 
your project still has a linters errors, beside of this your project has a clean  html and script code with great design.

**Comment by ZohalMirkhel** [comment]
@WALWALASAFI, please take a look at my project. I believe I am done with making the website responsive. Look at all parts of the project and let me know if I have forgotten anything. However, I still have not worked on the show less button of the certificates page, linters error, and README.md file, kindly ignore those. Thank you!

**Comment by ZohalMirkhel** [comment]
Dear @WALWALASAFI, did you check my project? Thank You!

**Comment by ZohalMirkhel** [comment]
@ahzamir, Hello sir! Please take a look at my project. The UX/UI designers are yet to add pictures to the pop-up in mobile view and the footer, which they will be doing today. However, I submitted my project to avoid missing another deadline.

**Comment by ZohalMirkhel** [comment]
Hi @ahzamir, could you please take a look at my project when you get a chance? I’d really appreciate your feedback. Thanks!

**Comment by ahzamir** [comment]
@ZohalMirkhel 

Well, Done Zohal Mirkhel your project functionality is perfect, but in order to give feedback for the design and as a part of the requirements you need to add the Figma link on your README.md file and also add it to this comments replay.

**Comment by ZohalMirkhel** [comment]
Sure sir @ahzamir, here is the link for the Figma design, https://www.figma.com/design/n2sJXMUcTX7VmMfxjVnrxQ/Untitled?node-id=0-1&t=XDZgs2yMGJzUZKsS-0 
Thank you!

File: style.css
Comment (ahzamir) [inline style.css:1]: ### Home Page:
> Overall impressive, perfect with the functionality just some minor Design issues:

1. The Header (Navbar):
  - Padding issue (Has more padding than the Figma)
  - Font Issue (Is not same as the Figma)
2.  Social Media (links icons):
  - Sizing (Should be bigger)
  - Placing (Should be centered on the page)
  - Implement for both Screen sizings
3. Contact Me Button:
  - Margin Top (Has less margin than the Figma)
  - Font Weight (Is not same as the Figma) - Implement for both screen sizings
 - This button should be a link to the Contact page (The contact page should be a separate page same as other pages, not a pop-up modal)

  - Your Design:
  
image

  - Figma Design:
  
image

Comment (ahzamir) [inline style.css:1]: ### About Me Page:
> Overall impressive, perfect with the functionality just some minor Design issues:

1. Heading (About Me):
  - The background color (Is not the same as the Figma)
  - Implement for both screen sizings
2. The About Me Page text:
  - Line Height (Line spacing is not the same as the Figma design)
  - Implement for both screen sizing
  
  
  - Your Design:
  
image

  - Figma Design:
  
image

Comment (ahzamir) [inline style.css:1]: ### Education Page:
> Overall impressive, perfect with the functionality just some minor Design issues:

  - Implement the same feedback for the `Heading`, `Text`, and `Button`:
  
  - Your Design:
  
image

  - Figma Design:
  
image

Comment (ahzamir) [inline style.css:1]: ### Projects Page:
> Overall impressive, perfect with the functionality just some minor Design issues, Impressive design functionality implemented for the see more project icon ⭐⭐⭐:

  - Implement the same feedback for the `Heading`, `Text`, and `Button` for both screen sizings.
  - Project's name are different from the Figma Design
  - Change your icon and make it exactly the same as the Figma design.
  - Padding issue (Mobile version).
  - Project's image, text and ... are smaller on figma design

  - Your Design:
  
image
image


  - Figma Design:
  
image
image

File: script.js
Comment (ahzamir) [inline script.js:1]: ### Certifications:
> Impressive design implemented, just implement the same feedback for the design as other pages:

- You need to add the certification links for every certification as it is part of the requirement:
  - You can add it on the image or certification name or separately.

File: style.css
Comment (ahzamir) [inline style.css:1]: ### Skills Page:
> Overall impressive, perfect with the functionality just some minor Design issues, Impressive design functionality implemented for the see more project icon ⭐⭐⭐:

  - Implement the same feedback for the `Heading` for both sizings.

Comment (ahzamir) [inline style.css:1]: ### Contact Page:
> This page should be a separated page or section the same as the other pages (Remove pop-up and add a separate page for the contact):

- Text Sizing (Is different from the Figma)
- Contact Page text padding (Is different from the Figma)
- The Footer background and text typography are not the same as the Figma Design.
- The mobile version is fine but has some little differences from the Figma

  - Your Design:
  
image

  - Figma Design:
    
image

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @ZohalMirkhel 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Well done @ZohalMirkhel , Your hard work and passion served on this project resulted such an amazing project, but still your project has some minor issues you need to work on in order to get the final approval:
  - Check the comments below
  
# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
❌ You have followed and completed the JS Capstone Project and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive
❌Your design is same as the Figma Designers Design

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

File: README.md
Comment (ahzamir) [inline README.md:83]: - Well done, your README.md file is very great  just implement these 2 changes:
  - At least add one social profile link for the UI/UX Designers besides their name.
  - Add the link of the Figma Design on your README

Comment (ahzamir) [inline README.md]: - Well done, your README.md file is very great  just implement these 2 changes:
  - At least add one social profile link for the UI/UX Designers besides their name.
  - Add the link of the Figma Design on your README

**Review by ahzamir — CHANGES_REQUESTED** [review]

**Comment by ZohalMirkhel** [comment]
Dear @ahzamir,
I believe I have completed the project to the best of my ability, incorporating all the feedback you provided. I also reviewed my work with my partner, and she confirmed that I had addressed all the feedback. I look forward to your response!

Thank You!

File: style.css
Comment (ahzamir) [inline style.css:1]: - The button is linked to contact-popup but your contact page ID is #contact please solve this as your button does not have any function for now:

image

- Your contact form has lots of space on its top please solve this and instead add some margin bottom as your contact does not have any margin-bottom
- Your icons are too small compared to the Figma design please add some height and width to them
- When the user types in the input field it's white and hard to read, please make them black and give them some font-weight to make it look clear.

image

**Review by ahzamir — APPROVED** [review]
### Hi @ZohalMirkhel 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.
- Congradulations Zohal, you have implemented all of the feedbacks and worked hard on this amazing project there is nothing to say other then merge your branch:
  - Before you merge your branch please have a look on 2 comments below and implement them.
- Well Done great design Implemented. 🎯🎉👍

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF


 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the JS Capstone project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive
✔️Your Design is the same as the Figma design

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by ZohalMirkhel** [comment]
@ahzamir, thank you so much, sir. I brought the changes you suggested and then merged my branch. 
Thank you again!

**Pull request merged**
